src/components/Navigation.js

```diff
@@ -7,13 +7,14 @@ export default function Navigation() {
       <LinkButton to="/">All Drinks</LinkButton>
       <LinkButton to="/favorites">Favorites</LinkButton>
       <LinkButton to="/myDrinks">My Drinks</LinkButton>
+      <LinkButton to="/randomDrink">Random</LinkButton>
     </NavBar>
   );
 }
 
 const NavBar = styled.nav`
   display: grid;
-  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
you could use a repeat here
```suggestion
  grid-template-columns: repeat(4,1fr);
```